MetaMask/metamask-extension
# Security and Privacy Settings Re-org

## Explanation

Reorganize settings so that all privacy related settings are easily discoverable, understood and toggleable.

Some settings (e.g Enhanced Token Detection) are in advanced tab, they should be moved to Security and Privacy Tab
There should also be a toggle to switch off/on all security and privacy toggles.

* Fixes #16727

## Screenshots/Screencaps



### Before
<img width="607" alt="Screenshot 2022-12-01 at 16 05 10" src="https://user-images.githubusercontent.com/44811/205048829-5763a27d-3265-4b23-acde-99f4c8bd9d9f.png">

### After
<img width="609" alt="Screenshot 2022-12-01 at 16 05 22" src="https://user-images.githubusercontent.com/44811/205048784-95b14952-adf1-40b2-b499-62c4e68c311c.png">

## Manual Testing Steps

1. Launch MetaMask
2. Open Settings
3. Make sure all security and privacy settings (see #16727) are under the Security and Privacy Tab
4. Make sure there's a checkbox on the tab to switch off/on all checkboxes in the tab

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
